### PR TITLE
KEB catalog to allow machine type update

### PR DESCRIPTION
**Pull request by wozniakjan**
**Description**
KEB catalog changes to support machine-type update.

[catalog.sorted.old.json.txt](https://github.com/kyma-project/control-plane/files/10569796/catalog.sorted.old.json.txt)
[catalog.sorted.new.json.txt](https://github.com/kyma-project/control-plane/files/10569797/catalog.sorted.new.json.txt)

<details>
  <summary>catalog diff</summary>

```diff
163a164
>                     "machineType",
186a188,214
>                     "machineType": {
>                       "_enumDisplayName": {
>                         "m5.12xlarge": "m5.12xlarge (48vCPU, 192GB RAM)",
>                         "m5.2xlarge": "m5.2xlarge (8vCPU, 32GB RAM)",
>                         "m5.4xlarge": "m5.4xlarge (16vCPU, 64GB RAM)",
>                         "m5.8xlarge": "m5.8xlarge (32vCPU, 128GB RAM)",
>                         "m5.xlarge": "m5.xlarge (4vCPU, 16GB RAM)",
>                         "m6i.12xlarge": "m6i.12xlarge (48vCPU, 192GB RAM)",
>                         "m6i.2xlarge": "m6i.2xlarge (8vCPU, 32GB RAM)",
>                         "m6i.4xlarge": "m6i.4xlarge (16vCPU, 64GB RAM)",
>                         "m6i.8xlarge": "m6i.8xlarge (32vCPU, 128GB RAM)",
>                         "m6i.xlarge": "m6i.xlarge (4vCPU, 16GB RAM)"
>                       },
>                       "enum": [
>                         "m5.12xlarge",
>                         "m5.2xlarge",
>                         "m5.4xlarge",
>                         "m5.8xlarge",
>                         "m5.xlarge",
>                         "m6i.12xlarge",
>                         "m6i.2xlarge",
>                         "m6i.4xlarge",
>                         "m6i.8xlarge",
>                         "m6i.xlarge"
>                       ],
>                       "type": "string"
>                     },
374a403
>                     "machineType",
397a427,445
>                     "machineType": {
>                       "_enumDisplayName": {
>                         "Standard_D16_v3": "Standard_D16_v3 (16vCPU, 64GB RAM)",
>                         "Standard_D32_v3": "Standard_D32_v3 (32vCPU, 128GB RAM)",
>                         "Standard_D48_v3": "Standard_D48_v3 (48vCPU, 192GB RAM)",
>                         "Standard_D4_v3": "Standard_D4_v3 (4vCPU, 16GB RAM)",
>                         "Standard_D64_v3": "Standard_D64_v3 (64vCPU, 256GB RAM)",
>                         "Standard_D8_v3": "Standard_D8_v3 (8vCPU, 32GB RAM)"
>                       },
>                       "enum": [
>                         "Standard_D16_v3",
>                         "Standard_D32_v3",
>                         "Standard_D48_v3",
>                         "Standard_D4_v3",
>                         "Standard_D64_v3",
>                         "Standard_D8_v3"
>                       ],
>                       "type": "string"
>                     },
575a624
>                     "machineType",
598a648,656
>                     "machineType": {
>                       "_enumDisplayName": {
>                         "Standard_D4_v3": "Standard_D4_v3 (4vCPU, 16GB RAM)"
>                       },
>                       "enum": [
>                         "Standard_D4_v3"
>                       ],
>                       "type": "string"
>                     },
945a1004
>                     "machineType",
968a1028,1044
>                     "machineType": {
>                       "_enumDisplayName": {
>                         "n2-standard-16": "n2-standard-16 (16vCPU, 64GB RAM)",
>                         "n2-standard-32": "n2-standard-32 (32vCPU, 128GB RAM)",
>                         "n2-standard-4": "n2-standard-4 (4vCPU, 16GB RAM)",
>                         "n2-standard-48": "n2-standard-48 (48vCPU, 192B RAM)",
>                         "n2-standard-8": "n2-standard-8 (8vCPU, 32GB RAM)"
>                       },
>                       "enum": [
>                         "n2-standard-16",
>                         "n2-standard-32",
>                         "n2-standard-4",
>                         "n2-standard-48",
>                         "n2-standard-8"
>                       ],
>                       "type": "string"
>                     },
1224a1301
>                     "machineType",
1247a1325,1335
>                     "machineType": {
>                       "_enumDisplayName": {
>                         "g_c4_m16": "g_c4_m16 (4vCPU, 16GB RAM)",
>                         "g_c8_m32": "g_c8_m32 (8vCPU, 32GB RAM)"
>                       },
>                       "enum": [
>                         "g_c4_m16",
>                         "g_c8_m32"
>                       ],
>                       "type": "string"
>                     },
1436a1525
>                     "machineType",
1458a1548,1574
>                     },
>                     "machineType": {
>                       "_enumDisplayName": {
>                         "m5.12xlarge": "m5.12xlarge (48vCPU, 192GB RAM)",
>                         "m5.2xlarge": "m5.2xlarge (8vCPU, 32GB RAM)",
>                         "m5.4xlarge": "m5.4xlarge (16vCPU, 64GB RAM)",
>                         "m5.8xlarge": "m5.8xlarge (32vCPU, 128GB RAM)",
>                         "m5.xlarge": "m5.xlarge (4vCPU, 16GB RAM)",
>                         "m6i.12xlarge": "m6i.12xlarge (48vCPU, 192GB RAM)",
>                         "m6i.2xlarge": "m6i.2xlarge (8vCPU, 32GB RAM)",
>                         "m6i.4xlarge": "m6i.4xlarge (16vCPU, 64GB RAM)",
>                         "m6i.8xlarge": "m6i.8xlarge (32vCPU, 128GB RAM)",
>                         "m6i.xlarge": "m6i.xlarge (4vCPU, 16GB RAM)"
>                       },
>                       "enum": [
>                         "m5.12xlarge",
>                         "m5.2xlarge",
>                         "m5.4xlarge",
>                         "m5.8xlarge",
>                         "m5.xlarge",
>                         "m6i.12xlarge",
>                         "m6i.2xlarge",
>                         "m6i.4xlarge",
>                         "m6i.8xlarge",
>                         "m6i.xlarge"
>                       ],
>                       "type": "string"

```

</details>

**Related issue(s)**

depends on: https://github.com/kyma-project/control-plane/pull/2447
closes: https://github.com/kyma-project/control-plane/issues/2446

### CODE REVIEW

File: components/kyma-environment-broker/internal/broker/testdata/aws-schema-additional-params.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "administrators": {
       "description": "Specifies the list of runtime administrators",
       "items": {
-          "type": "string"
+        "type": "string"
```
Comment (wozniakjan) [inline components/kyma-environment-broker/internal/broker/testdata/aws-schema-additional-params.json:17]: this unrelated whitespace diff appeared after getting all `testdata` fixtures in sync with `/catalog` endpoint output, now the indent is consistent 2 spaces per each level.

**Review by PK85 — APPROVED** [review]
